tests/forte/data/data_store_test.py

```diff
@@ -21,7 +21,7 @@
 from typing import Optional, Dict
 from dataclasses import dataclass
 from forte.data.data_store import DataStore
-from forte.data.ontology.top import Annotation, Generics
+from forte.data.ontology.top import Annotation, Generics, Group, Link
```

Comment:
I saw some test cases already imported `Annotation` here, So do we need to import these locally in the `test_is_subclass` function?

Reply:
Let's try to move all of them to local since importing here will change some test behaviors.